walkthroughs/howto-mutual-tls-file-provided/infrastructure/ecs-service.yaml

```diff
@@ -203,15 +201,15 @@ Resources:
           Environment:
             - Name: 'APPMESH_VIRTUAL_NODE_NAME'
               Value: !Sub 'mesh/${AppMeshMeshName}/virtualGateway/gateway-vgw'
-            - Name: 'APPMESH_PREVIEW'
-              Value: '1'
             - Name: CERTIFICATE_NAME
               Value: 'gateway'
             - Name: ENVOY_LOG_LEVEL
               Value: 'debug'
 
   GatewayService:
     Type: 'AWS::ECS::Service'
+    DependsOn:
+      - WebTargetGroup
```

Comment:
I feel like this should be implicit because of line 235 but I had this resource fail to create (and thus the stack) because the WebTargetGroup DNE.

```
232     LoadBalancers:
233       - ContainerName: envoy
234         ContainerPort: 9080
235         TargetGroupArn: !Ref WebTargetGroup
```

Reply:
Went back to the events to see the exact error. The actual dependency that was a problem was `WebTargetGroup` --> `PublicLoadBalancer`. There's already a depends on statement so I think this was spurious. 

```
Invalid request provided: CreateService error: The target group with targetGroupArn arn:aws:elasticloadbalancing:us-west-2:622196456088:targetgroup/AppMeshMutualTLSExample-web2/4655ef29001b35b2 does not have an associated load balancer.
```